test/sample-test/sample_test_launcher.py

```diff
@@ -120,6 +119,9 @@ def _compile(self):
         if 'run_pipeline' in raw_args.keys():
           self._run_pipeline = raw_args['run_pipeline']
 
+      if self._run_pipeline:
+        nb_params['experiment_name'] = self._test_name + '-test'
```

Comment:
Shouldn't the config override the test infra default?

Reply:
Actually I would like to propose to make experiment_name parameterized but not configurable.
The reason is that IMO the experiment name should not have any impact on the functionalities of a KFP sample, and letting the user configure that will put us under the risk of potential name conflict. For example, if two user-contributed sample specified identical experiment names, it might be a problem when we list all the runs under the experiment.

Reply:
Good point. I agree.